rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -929,13 +929,12 @@ public byte[] getRetiringFederatorPublicKeyOfType(Object[] args) throws VMExcept
     public Long getRetiringFederationCreationTime(Object[] args) {
         logger.trace("getRetiringFederationCreationTime");
 
-        Instant retiringFederationCreationTime = bridgeSupport.getRetiringFederationCreationTime();
-
-        if (retiringFederationCreationTime == null) {
-            // -1 is returned when no retiring federation
-            return -1L;
-        }
+        Optional<Instant> retiringFederationCreationTime = bridgeSupport.getRetiringFederationCreationTime();
+        return retiringFederationCreationTime.map(this::getEpochTimeBasedOnActivation)
+            .orElse((long) FederationChangeResponseCode.FEDERATION_NON_EXISTENT.getCode());
+    }
 
+    private Long getEpochTimeBasedOnActivation(Instant retiringFederationCreationTime) {
```

Comment:
```suggestion
    private Long getFederationCreationTiimeEpochBasedOnActivation(Instant federationCreationTime) {
```
Make it clear that it applies to federation creation time, not to any epoch. Can/should this be re-used when getting active federation creation time?

Reply:
marcos is right, this same logic is being applied for `getFederationCreationTime`